src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -502,76 +486,157 @@ export const BorrowInput = ({
     lendingQuoteError?.message,
   ])
 
+  const handleBorrowSubmit = useCallback(() => {
+    return !!isUnsafeQuote ? setShouldShowWarningAcknowledgement(true) : onSubmit()
```

Comment:
Casting is unnecessary since this is a boolean or `null` so truthy `isUnsafeQuote` will always be `true`

Reply:
Indeed! Addressed in https://github.com/shapeshift/web/pull/6647/commits/916ecb47d6f7964089d596f97a9406fbe95c1d43.